src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyNameFormatter.cs

```diff
@@ -101,24 +103,24 @@ private static void AppendQuoted(this StringBuilder sb, string s)
                     case '=':
                     case '\'':
                     case '"':
-                        sb.Append('\\');
+                        vsb.Append('\\');
                         break;
                     case '\t':
-                        sb.Append("\\t");
+                        vsb.Append("\\t");
```

Comment:
can it use the char overload (I assume it's faster?)

Reply:
Sure

Reply:
Wait, no, that's not a char :-)

Reply:
oops 😨